components/espace-agent-components/documents/document-wall.tsx

```diff
@@ -0,0 +1,106 @@
+import { PropsWithChildren } from 'react';
+import routes from '#clients/routes';
+import ButtonMonComptePro from '#components-ui/button-mon-compte-pro';
+import { Section } from '#components/section';
+import { FullTable } from '#components/table/full';
+import { EAdministration } from '#models/administrations/EAdministration';
+import { IUniteLegale } from '#models/index';
+
+const DocumentWall: React.FC<
+  PropsWithChildren<{
+    isVisible: boolean;
+    title: string;
+    id: string;
+    uniteLegale: IUniteLegale;
+  }>
+> = ({ uniteLegale, id, title, isVisible = false, children }) => {
+  if (!isVisible) {
+    return (
```

Comment:
It's considered good practice to return early with the simplest case, to avoid unnecessary indentation. 
```suggestion
  if (isVisible) {
    return children
  }
  return (
```